packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js

```diff
@@ -98,7 +98,13 @@ export function buildRootObject(vatPowers) {
       done() {
         return doneP;
       },
-      upgrade(bundlecap, vatParameters) {
+      upgrade(bundlecap, options) {
```

Comment:
`options` should default to `{}`, so existing code that isn't using it don't need to change.

Reply:
Good catch!

I guess omitting `vatParameters` in the upgrade should get you the same value as you'd get omitting it in the original `createVat`. That *ought* to be `undefined`, but now that I'm looking at the code, I'm not sure that's what happens. I'll add a test.